src/main/java/com/google/cloud/teleport/templates/TextToBigQueryStreaming.java

```diff
@@ -16,6 +16,8 @@
 
 package com.google.cloud.teleport.templates;
 
+import com.google.api.client.json.JsonFactory;
```

Comment:
The only additional imports needed would be:
com.google.api.client.json.JsonFactory
and
org.apache.beam.sdk.util.Transport